modules/gltf/gltf_document.cpp

```diff
@@ -4872,12 +4872,18 @@ Error GLTFDocument::_serialize_animations(Ref<GLTFState> p_state) {
 		return OK;
 	}
 	for (int32_t player_i = 0; player_i < p_state->animation_players.size(); player_i++) {
-		List<StringName> animation_names;
 		AnimationPlayer *animation_player = p_state->animation_players[player_i];
-		animation_player->get_animation_list(&animation_names);
-		if (animation_names.size()) {
-			for (int animation_name_i = 0; animation_name_i < animation_names.size(); animation_name_i++) {
-				_convert_animation(p_state, animation_player, animation_names[animation_name_i]);
+		List<StringName> animation_library_names;
+		animation_player->get_animation_library_list(&animation_library_names);
+		for (StringName library_name : animation_library_names) {
```

Comment:
Why was the original code incorrect? `get_animation_list` still exists and should return the full list of animations?

Reply:
animation_set contains animations from all libraries with /. I see you already updated the PR to reflect that `get_animation_list` works